.travis.yml

```diff
@@ -71,6 +73,15 @@ matrix:
           packages: ['gcc-7', 'g++-7']
           sources: &sources
             - ubuntu-toolchain-r-test
+            
+    - env: PLATFORM=Unix COMPILER=gcc-8
+      compiler: gcc
+      os: linux
+      addons:
```

Comment:
gcc-7 could be replaced with gcc-8. I don't think we need to cover many/all compiler versions.

Reply:
okay